clouddriver-sql/src/main/kotlin/com/netflix/spinnaker/config/SqlConfiguration.kt

```diff
@@ -100,6 +100,7 @@ class SqlConfiguration {
   }
 
   @Bean
+  @ConditionalOnExpression("\${sql.read-only:false} == false")
```

Comment:
And this fixes a separate problem with this agent wiring up - shouldn't be run if clouddriver is in readonly mode.

Reply:
Is this needed, isn't this taken into account here:
https://github.com/spinnaker/clouddriver/blob/master/cats/cats-sql/src/main/kotlin/com/netflix/spinnaker/config/SqlCacheConfiguration.kt#L181 ?

Reply:
The agent would still be created, which would be unnecessary. It won't run, but it'd also just be sitting in memory needlessly.

Reply:
gotcha, makes sense